database/dataGen.js

```diff
@@ -0,0 +1,49 @@
+const faker = require('faker');
+const fs = require('fs');
+
+const getRandomInt = (min, max) => {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * ((max - min) + 1)) + min;
+};
+
+const randomDate = () => {
+  const day = JSON.stringify(getRandomInt(1, 30));
+  const month = () => {
+    let monthNumber = getRandomInt(1, 12);
+    if (monthNumber < 10) {
+      monthNumber = '0' + JSON.stringify(monthNumber);
```

Comment:
It's seems like you need the JSON.stringify structure here, but if not you can simply add monthNumber to a string without invoking a function.